### ISSUE TITLE
The team @go-maintainers does not exist in bazel-contrib

### ISSUE THREAD

**Issue by malt3** [bazel-contrib/rules_go#4162]
This team has to be copied from the old org (unless I just can't see it)

**Comment by fmeum** [comment]
@alexeagle Are you able to make @bazel-contrib/go-maintainers truly public (rather than just org public)?

**Comment by alexeagle** [comment]
Sorry, no I don't see any settings I can change for this.

Note that each individual member can choose whether they'd like their membership to be public-visible:
https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility

Maybe @meteorcloudy can see whether bazelbuild has some relevant setting I'm not seeing that makes the two orgs behave differently.

**Comment by meteorcloudy** [comment]
Yeah, this is a limitation with GitHub, we have the same problem for the `@bazelbuild/triage` and `@bazelbuild/bcr-maintainers`, they only works for org memembers.
